extensions/query-store/.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+	"parserOptions": {
+		"project": "./extensions/query-store/tsconfig.json"
+	},
+	"rules": {
+		"@typescript-eslint/no-floating-promises": [
+			"error",
+			{
+				"ignoreVoid": true
+			}
+		],
+		"@typescript-eslint/explicit-function-return-type": ["off"]
```

Comment:
But why is this off? You should enable it now while there's nothing here so anything you add is fixed immediately.

Reply:
So good thing this was brought up...these rules are supposed to be enabled by default for anything in the extensions folder but that turned out to have been removed in a previous VS Code merge. PR out for bringing it back : https://github.com/microsoft/azuredatastudio/pull/23271

So for this just remove the line completely - it'll default to error in that case.